app/src/main/java/org/oppia/android/app/testing/SplashTestActivity.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.android.app.testing
+
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+class SplashTestActivity : InjectableAppCompatActivity() {
```

Comment:
Add a KDoc

Reply:
Done

Reply:
Would it be better to name this `PlatformParametersTestActivity` (and associated presenter & tests)? Since it doesn't matter that it's faking the `SplashActivity`, but it does matter that it's needed to test PlatformParameters.

Reply:
As all the PlatformParameter related task will be triggered by `SplashActivity` only thats why I kept this name for the FakeActivity to represent where is the actual change required. Although I was thinking to rename the `SplashTestActivityTest` to `PlatformParameterIntegrationTest` to represent that we are actually testing PlatformParameter architecture with the help of Fake Activity (SplashTestActivity)

Reply:
Hm okay I think it's fine to leave this file but I do think that the test should be renamed to `PlatformParameterIntegrationTest` or `PlatformParameterTest` since we're not really testing a fake activity, we're testing that we properly fetch parameters. I would also add to the comment that it's a fake SplashActivity since that is what fetches the parameter, something like `A test activity to verify the injection of [PlatformParameter] in the [SplashActivity]. This test activity is used in integration tests for platform parameters.`.

Reply:
Done